### ISSUE TITLE
Failed to execute Command Line Tool

### ISSUE THREAD

**Issue by Eldorado234** [desktop/desktop#2353]
### Description

After installing the command line tool it cannot be executed on my system:
```
Failed to execute process '/usr/local/bin/github'. Reason:
exec: Exec format error
The file '/usr/local/bin/github' is marked as an executable but could not be run by the operating system.
```

### Version

**GitHub Desktop version:** 0.7.0

**OS version:** macOS 10.12.5

### Steps to Reproduce

1. Install the command line tools via the GitHub Desktop UI
1. Execute `github` on the command line
1. Get the error above

**Expected behavior:** Seeing a help text printed which tells me what can be accomplished with the tool

**Actual behavior:** See above

**Reproduces how often:** Every time on my machine

### Logs

Nothing interesting in there.

**Comment by Eldorado234** [comment]
Ah, I guess I found the reason: there is no shebang present and I use the fish shell. Adding `#!/bin/sh` to the top of the `github` script solves the issue.

**Comment by shiftkey** [comment]
@Eldorado234 this has been fixed in #2299 and will be available in the next update!
